core/src/main/java/org/apache/calcite/rel/rules/ProjectRemoveRule.java

```diff
@@ -88,7 +88,6 @@ public static boolean isTrivial(Project project) {
         project.getInput().getRowType());
   }
 
-  @Deprecated // to be removed before 1.5
   public static boolean isIdentity(List<? extends RexNode> exps,
       RelDataType childRowType) {
```

Comment:
I don't understand what you want to do. Could you explain more about it?

Reply:
I am only testing the continuous testing failure with a trivial change. It seems irrelevant to my patch. I've just pushed the real change.